wallet/wallet-cli-lib/src/commands/mod.rs

```diff
@@ -1104,6 +1116,22 @@ impl CommandHandler {
                 Ok(Self::handle_mempool_tx_status(status))
             }
 
+            WalletCommand::ShowSeedPhrase => {
+                let phrase = controller_opt
+                    .as_mut()
+                    .ok_or(WalletCliError::NoWallet)?
+                    .seed_phrase()
+                    .map_err(WalletCliError::Controller)?;
+
+                let msg = if let Some(phrase) = phrase {
+                    format!("The saved seed phrase is \"{}\"", phrase)
+                } else {
+                    "No saved seed phrase for this wallet.".into()
```

Comment:
No saved seed phrase for this wallet. This was your choice when you created the wallet as a security option. Make sure not to lose this wallet file.

Reply:
changed